core/db.py

```
@@ -112,14 +235,53 @@ def session_scope() -> Generator[Session, None, None]:
+        await session.close()
+
+
 def init_db() -> None:
     """RU: Создаёт схему таблиц для зарегистрированных моделей (например, при старте).
 
     EN: Creates database schema for all registered models (used during startup).
     """
 
     # Import models lazily so Base metadata is populated before create_all is called.
-    import core.models  # noqa: F401  # pylint: disable=unused-import
+    import core.models  # pylint: disable=unused-import
```

Flake8 (7.3.0)

[error] 'core.models' imported but unused

(F401)